cocos2d/core/platform/CCSys.js

```diff
@@ -801,6 +806,13 @@ function initSys () {
 
         var currLanguage = nav.language;
         currLanguage = currLanguage ? currLanguage : nav.browserLanguage;
+
+        /**
+         * Indicate the current full language of the running system
```

Comment:
.... Examples of valid language codes include "zh-tw", "en", "en-us", "fr", "fr-fr", "es-es", etc.